client/azure_client_set.go

```diff
@@ -44,6 +45,20 @@ type AzureClientSet struct {
 	VnetPeeringClient *network.VirtualNetworkPeeringsClient
 }
 
+func init() {
+	// ONE DOES NOT SIMPLY RETRY ON HTTP 429.
+	for i, sc := range autorest.StatusCodesForRetry {
```

Comment:
It took me a couple of minutes to understand what this was doing. Can we wrap it in a function called `remove429FromRetries` or whatever? Or a code comment `Remove 429 from codes to retry`?

Reply:
I'll write a `removeElementFromSlice()` function so it's more readable :slightly_smiling_face: